libs/design-system/src/core/Button.tsx

```diff
@@ -127,6 +127,17 @@ export const buttonStyles = cva(
           'hover:text-gray-1000 hover:dark:text-graydark-1000',
           'text-gray-700 dark:text-graydark-800',
         ],
+        amber2: [
```

Comment:
Lets keep the action button colors semantic, ie, primary green, secondary gray, danger destructive red, etc.

I see where this comes from: because we match the primary accent green with the Sia network green, it makes buttons like this feel a little inconsistent on zen. But if we want to tackle this, I think the full fix here is to either:
- De-emphasize the relationship between current network and button colors by changing the primary accent color. I think I prefer this.
- Investigate and implement a dynamic button accent color pattern that changes per network and still makes sense with the semantic colors.
In both cases this would apply to all our apps.